cmd/tracee-rules/main.go

```diff
@@ -182,6 +185,19 @@ func main() {
 				Name:  "list",
 				Usage: "print all available rules",
 			},
+			&cli.StringFlag{
```

Comment:
maybe we could have only one flag, and the rest as options like we do in tracee-ebpf, this would decrease the number of flags added withing one scope `--forward`, eg:

```
tracee-ebpf  --capture write=/usr/bin/* --capture write=/etc/* 
```

https://github.com/aquasecurity/tracee/blob/main/cmd/tracee-ebpf/flags/capture.go

Reply:
Yeah much better, I just followed what was there but it didn't sit well with me.